laravel.md

```diff
@@ -0,0 +1,46 @@
+# Laravel
+
+**Prerequisite knowledge:** php, oop, mysql, html, frameworks, mvc
```

Comment:
I would add composer here although on the other hand you can use laravel using the homestead and then everything is set up for you.

Reply:
In one of those levels I would mention dependency management anyway.